### PR TITLE
[admin] Added more info in RadiusProfileAdmin list view.

### PR DESCRIPTION
**Pull request by R9295**
implements #136

### CODE REVIEW

File: django_freeradius/base/admin.py [hunk 1]
```diff
@@ -206,7 +206,26 @@ def get_readonly_fields(self, request, obj=None):
...
+        # returns daily session limit in hours
+        if obj.daily_session_limit:
+            return round(obj.daily_session_limit / 3600, 2)
+    get_daily_session_limit.short_description = 'daily session limit(hours)'
+
+    def get_daily_bandwidth_limit(self, obj):
+        # returns daily bandwidth limit in megabytes
+        if obj.daily_bandwidth_limit:
+            return int(obj.daily_bandwidth_limit / 1000 ** 2)
+    get_daily_bandwidth_limit.short_description = 'daily bandwidth limit(MB)'
```
Comment (rohithasrk) [inline django_freeradius/base/admin.py]: Add a space after limit in short description, i.e., `daily bandwidth limit (MB)`

File: django_freeradius/base/admin.py [hunk 2]
```diff
@@ -206,7 +206,26 @@ def get_readonly_fields(self, request, obj=None):
...
+
+    def get_max_all_time_limit(self, obj):
+        # returns max all time limit in hours
+        if obj.max_all_time_limit:
+            return round(obj.max_all_time_limit / 3600, 2)
+    get_max_all_time_limit.short_description = 'maximum all time limit(hours)'
```
Comment (rohithasrk) [inline django_freeradius/base/admin.py]: Same as above.

File: django_freeradius/base/admin.py [hunk 3]
```diff
@@ -206,7 +206,26 @@ def get_readonly_fields(self, request, obj=None):
...
 class AbstractRadiusProfileAdmin(TimeStampedEditableAdmin):
-    pass
+    list_display = ['name', 'get_daily_session_limit', 'get_daily_bandwidth_limit', 'max_all_time_limit',
+                    'created', 'modified']
+
+    def get_daily_session_limit(self, obj):
+        # returns daily session limit in hours
+        if obj.daily_session_limit:
+            return round(obj.daily_session_limit / 3600, 2)
+    get_daily_session_limit.short_description = 'daily session limit(hours)'
```
Comment (rohithasrk) [inline django_freeradius/base/admin.py]: Space after limit and before the parenthesis. `daily session limit (hours)`

File: django_freeradius/base/admin.py [hunk 4]
```diff
@@ -206,7 +206,26 @@ def get_readonly_fields(self, request, obj=None):
 
 
 class AbstractRadiusProfileAdmin(TimeStampedEditableAdmin):
-    pass
+    list_display = ['name', 'get_daily_session_limit', 'get_daily_bandwidth_limit', 'max_all_time_limit',
```
Comment (rohithasrk) [inline django_freeradius/base/admin.py]: @R9295 I think it should be `get_max_all_time_limit` and not `max_all_time_limit`. Writing tests would have been useful here.

**Comment by rohithasrk** [comment]
Also would be great if there was a space after the parenthesis in the commit message. `[admin] Added more info in RadiusProfileAdmin list view`

File: django_freeradius/base/admin.py (hunk 3 above)
Comment (nemesisdesign) [reply django_freeradius/base/admin.py]: it should be `_('daily session limit (hours)')`, human readable strings needs to be internationalization ready for the future

File: django_freeradius/base/admin.py (hunk 1 above)
Comment (nemesisdesign) [reply django_freeradius/base/admin.py]: same here

File: django_freeradius/base/admin.py (hunk 2 above)
Comment (nemesisdesign) [reply django_freeradius/base/admin.py]: same here

File: django_freeradius/base/admin.py (hunk 3 above)
Comment (rohithasrk) [reply django_freeradius/base/admin.py]: @nemesisdesign is right. It slipped out of my mind yesterday.

File: django_freeradius/base/admin.py [hunk 5]
```diff
@@ -198,7 +198,26 @@ def get_readonly_fields(self, request, obj=None):
 
 
 class AbstractRadiusProfileAdmin(TimeStampedEditableAdmin):
-    pass
+    list_display = ['name', 'get_daily_session_limit', 'get_daily_bandwidth_limit', 'get_max_all_time_limit',
```
Comment (nemesisdesign) [inline django_freeradius/base/admin.py]: please list these vertically, experience taught me it's better (changing one item doesn't change the whole line and it's easier to track)

Comment (R9295) [reply django_freeradius/base/admin.py]: :+1:

**Comment by rohithasrk** [comment]
Thank you @R9295. Great job.

**Pull request merged**